### ISSUE TITLE
Ability to change archive password

### ISSUE THREAD

**Issue by ada-sk** [raivo-otp/ios-application#99]
**Is your feature request related to a problem? Please describe.**
I have forgotten my archive password and there is no way to change it

**Describe the solution you'd like**
The ability to change the password for new exports

**Describe alternatives you've considered**
Attempted changing passcode but it doesn't change the archive password

**Comment by alxgda** [comment]
I'm also interested in this

**Comment by Zazou49** [comment]
I set Raivo OTP 2 years ago, just to try it because I didn’t want use google Authenticator… but I fell in love with the App. 

Today I use Raivo everyday, I have more than 20 entries in the app and today I’m afraid to restore my iPhone because I don’t remember the master password (yes I was dumb).

My data are synced with iCloud provider but I don’t know if I restore the app from iCloud backup the master password will be required. 

So I would like to see a feature to change the master password :D

**Comment by bluto32** [comment]
Do you have a second smart phone? If so, you could try this:

- Install Raivo on the second phone with a new master password, but without cloud backup (in case things are left behind later).
- Swipe each Raivo token to the left on your first phone, select "QR code", and scan it with Raivo on your second phone.
- Check all tokens are present on your second phone.
- Reset Raivo on your first phone with your new master password.
- Transfer the tokens from your second phone back to the first phone by going through the same process.
- Check all tokens are present on your first phone.
- Uninstall Raivo from your second phone.

**Comment by AMlG0** [comment]
Yes I am also interested in this. I'm sure I was prompted to save the password, but like someone stated above, I wanted to try it out and ended up keeping it. I am scared I will be locked out of all my accounts if my phone breaks or I need to restore and I will pretty much be screwed. The only solution I could think of would be to literally make a brand new account and transfer everything again, but that seems completely ridiculous. Adding the ability to change the master password would be much appreciated.
